test/fixtures/transformation/es6-tail-call/try-catch/expected.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+(function f(n) {
+  if (n <= 0) {
+    return "foo";
+  }
+  try {
+    return f(n - 1);
+  } catch (e) {}
+})(1000000) === "foo";
+
+(function f() {

```

Comment:
Ah right, duh. Thanks for this, really excited to get this merged! Ping me when you think it's ready to be merged. Could also use an [exec.js](https://github.com/6to5/6to5/blob/master/test/fixtures/transformation/spec-typeof-symbol/basic/exec.js) test to be extra safe.